ecosystem/sep-0024.md

```diff
@@ -324,8 +324,8 @@ Request Parameters:
 
 | Name                          | Type                    | Description                                                                                                                                                                                                                                                                                                                                                                                                                      |
 | ----------------------------- | ----------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| `asset_code`                  | string                  | The code of the stellar asset the user wants to receive for their deposit with the anchor. The value passed must match one of the codes listed in the [/info](#info) response's deposit object.                                                                                                                                                                                                                                  |
-| `asset_issuer`                | string                  | (optional) The issuer of the stellar asset the user wants to receive for their deposit with the anchor. If asset_issuer is not provided, the anchor should use the asset issued by themselves as described in their TOML file.                                                                                                                                                                                                   |
+| `asset_code`                  | string                  | The code of the stellar asset the user wants to receive for their deposit with the anchor. The value passed must match one of the codes listed in the [/info](#info) response's deposit object. `native` is a special `asset_code` that represents the native XLM token.                                                                                                                                                                                                                   |
```

Comment:
Can you make sure the table is formatted? 
```
npx prettier  --write  ecosystem/sep-0024.md 
```

Reply:
Oops. Thanks!